api/api.py

```diff
@@ -0,0 +1,54 @@
+from fastapi import FastAPI, File, UploadFile
+from fastapi.middleware.cors import CORSMiddleware
+from google.cloud import vision
+import requests
+import os
+from os.path import join, dirname
+from dotenv import load_dotenv
+
+env_path = join(dirname(__file__),'.env')
+load_dotenv(dotenv_path=env_path)
+
+app = FastAPI()
+
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=["*"],  # Allows all origins
+    allow_credentials=True,
+    allow_methods=["*"],  # Allows all methods
+    allow_headers=["*"],  # Allows all headers
+)
+
+@app.get("/")
+def index():
+    return {"greeting": "Hello world"}
+
+@app.post("/uploadfile/")
+def create_upload_file(file: UploadFile = File(...)):
+    
+    urls = dummy_model(file.file)
+    return urls
+
+
+def dummy_model(file):
+    
+    GOOGLE_SEARCH_API_LINK='https://customsearch.googleapis.com/customsearch/v1'
+    api_key = os.getenv('API_KEY')
+    cx_key = os.getenv('CX_KEY')
```

Comment:
Are these keys something we can directly add to repo secrets? We'll need to test how to make it work on different laptops

Reply:
These are necessary only for the dummy model to work. I can send .env files (work with them was described by Ben during the lecture on Monday)

Reply:
👍